src/models.py

```diff
@@ -28,9 +31,39 @@ def exists(username=None, id=None):
 			return User.select().where(User.username == username).exists()
 
 
-models = [User]
+class Directory(BaseModel):
+	id = IntegerField()
+	parent = IntegerField()
+	depth = IntegerField()
+	name = CharField()
+
+	def get_full_path(self):
+		if self.id is not None:
+			if self.id is not 0:
+				return Directory.get(Directory.id == self.parent).get_full_path() + '\\' + self.name
+			else:
+				return self.name
+
+#maybe what a File will look like
+class File(BaseModel):
+	id = IntegerField()
+	directory_id = IntegerField()
+	name = CharField()
+	content = BlobField()
+	extension = CharField()
+
+models = [User, Directory, File]
+
 for model in models: 
+	model.drop_table(fail_silently=True)
 	model.create_table(fail_silently=True)
 
 if not User.select().where(User.username == 'root').exists():
-	User.create(username='root', password_hash=generate_password_hash('toor'))
\ No newline at end of file
+	User.create(username='root', password_hash=generate_password_hash('toor'), working_directory=1)
+
+#demo data for testing
+if not Directory.select().where(Directory.id == 0 or Directory.id == 1 or Directory.id == 2 or Directory.id == 3).exists():
```

Comment:
No reason to check for this if the tables were just dropped and rebuilt. All the tables will be empty.

Reply:
Done.